Source/EasyNetQ/Logging/ILogger.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+namespace EasyNetQ.Logging
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface ILogger
+    {
+        /// <summary>
+        ///     Log.
+        /// </summary>
+        /// <param name="logLevel">The log level</param>
+        /// <param name="messageFunc">The message function</param>
+        /// <param name="exception">The exception</param>
+        /// <param name="formatParameters">The format parameters</param>
+        /// <returns>A boolean.</returns>
```

Comment:
```suggestion
        /// <returns>A boolean value indicating if the provided level is enabled when used with messageFunc=null; otherwise the return value does not matter.</returns>
```